tests/components/color_picker.test.ts

```diff
@@ -51,12 +51,15 @@ describe("Color Picker buttons", () => {
     expect(onColorPicked).toHaveBeenCalledWith("");
   });
 
-  test("Click on '+' button toggle the custom color part", async () => {
+  test("Clicking on '+', custom section except custom added colors must toggle the custom color picker", async () => {
     await mountColorPicker();
-    await simulateClick(".o-color-picker-toggler");
-    expect(fixture.querySelector(".o-custom-selector")).toBeDefined();
-    await simulateClick(".o-color-picker-toggler");
-    expect(fixture.querySelector(".o-custom-selector")).toBeNull();
+    const elements = fixture.querySelectorAll(".o-color-picker-toggler");
+    elements.forEach(async (element) => {
```

Comment:
```suggestion
    for(const element of elements) {...}{
```
We try to avoid `foreach` in this repo, because some people find them ugly.

Side note, but when you're done with your changes, you should change the state of the task in Odoo, so we know we can do something for you. You should usually put the tak back in Rd Dev and green it (or in RD Test green if it was already tested, and you didn't change much)

Reply:
Ok. Applied suggested changes and changed it's state.